unittests/Swift/BuildKeyTests.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  BuildKeyTests.swift
+//  llbuildSwiftTests
+//
+//  Copyright © 2019 Apple Inc. All rights reserved.
+//
+
+import XCTest
+
+// The Swift package has llbuildSwift as module
+#if SWIFT_PACKAGE
+import llbuild
+import llbuildSwift
+#else
+import llbuild
+#endif
+
+private enum KeyTypeRepresentable {
+  case char(Character)
+  case number(UInt8)
+}
+
+extension KeyTypeRepresentable: ExpressibleByStringLiteral {
+  typealias StringLiteralType = String
+  
+  init(stringLiteral value: String) {
+    assert(value.count == 1)
+    self = .char(value.first!)
+  }
+}
+
+extension KeyTypeRepresentable: ExpressibleByIntegerLiteral {
+  init(integerLiteral value: UInt8) {
+    self = .number(value)
+  }
+}
+
+private extension Sequence where Element == KeyTypeRepresentable {
+  var asKey: KeyType {
+    compactMap { element in
+      switch element {
+      case .char(let char): return char.asciiValue
+      case .number(let num): return num
+      }
+    }
+  }
+}
+
+private extension String {
+  init?(_ key: KeyType) {
+    self.init(bytes: key, encoding: .utf8)
+  }
+  
+  var asKey: KeyType {
+    map { KeyTypeRepresentable.char($0) }.asKey
+  }
+}
+
+class BuildKeyTests: XCTestCase {
+
+  func testCommand() {
+    let command = BuildKey.Command(name: "foobar")
+    XCTAssertEqual(command.kind, .command)
+    XCTAssertEqual(command.name, "foobar")
+    XCTAssertEqual(command.keyData, "Cfoobar".asKey)
```

Comment:
I don't think we should be testing the raw keyData at this level of testing.

Reply:
Maybe not the specific encodings but that the keyData is bridged correctly is worth having a test I think

Reply:
Waiting on your feedback here before merging.